includes/class-wpcom-legacy-redirector.php

```diff
@@ -217,6 +217,24 @@ public static function get_url_hash( $url ) {
 		return md5( $url );
 	}
 
+	/**
+	 * Throws new error method
+	 *
+	 * @throws \Exception $message.
+	 *
+	 * @param [type] $code    Error code.
+	 * @param [type] $message Error message.
+	 * @return void
+	 */
+	public static function throw_error( $code, $message ) {
+		if ( class_exists( 'WP_Error' ) ) {
+			new WP_Error( $code, $message );
```

Comment:
Does this need a `return` at the start of the line?

Reply:
Yes, fixed now! Should we push this method to Utils class maybe and add a test for it?